ubtue_Revista electronica de ciencia penal y criminologia.js

```diff
@@ -0,0 +1,176 @@
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+const entriesXPath = '//p[@class="MsoFooter"]';
+const journalInfoXPath = '//p[@class="MsoNormal"]//span/text()';
+
+const articleNumberPrefix = '^\\s*\\d+-[a-z]?\\d+[a-z]?';
+const quotationMarks = '["”]';
+
+function detectWeb(doc, url) {
+	if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+	return false;
+}
+
+
+function getSearchResults(doc, checkOnly) {
+	let items = {};
+	let found = false;
+	let rows = ZU.xpath(doc, entriesXPath);
+	for (let row of rows) {
+		let title = ZU.trimInternal(row.textContent);
+		let anchor = row.querySelector('a');
+		let href = anchor ? anchor.getAttribute("href") : null;
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function getIssue(doc) {
+	let issueExpressions = ZU.xpath(doc, journalInfoXPath);
+	for (let exp of issueExpressions) {
+	   let issue = /Núm\.\s+(\d+).*/.exec(exp.nodeValue);
+	   if (issue)
+		   return issue[1];
+	}
+	// With the new layout there is seemingly no direct class
+	issueExpressions = ZU.xpath(doc, "//span[@style='font-size: 10pt;']");
+	for (let exp of issueExpressions) {
+		Z.debug(exp.innerText);
+		let issue = /Número\s+(\d+).*/.exec(exp.innerText);
+		if (issue)
+		    return issue[1];
+	}
+    return false;
+}
+
+
+function getYear(doc) {
+	let yearExpression = ZU.xpath(doc, journalInfoXPath);
+	for (let exp of yearExpression) {
+		let year =  /Núm\.\s+\d+\s*\((\d+)\)/.exec(exp.nodeValue);
+		if (year)
+			return year[1];
+	}
+    yearExpression = ZU.xpath(doc, "//span[@style='font-size: 10pt;']");
+	for (let exp of yearExpression) {
+		let issue = /Año\s+(\d+).*/.exec(exp.innerText);
+		if (issue)
+		    return issue[1];
+	}
+}
+
+
+function extractAuthors(entry) {
+	let authorRegex = new RegExp(articleNumberPrefix + '\\s+(.*),\\s+'
+		  + quotationMarks + '.*' + quotationMarks + '.*$');
+    let onelineInnerText = entry.innerText.replace(/[\r\n]+/gm, " ").replace(/\s\s+/gm, " ");
+	let authorPart = onelineInnerText.replace(authorRegex, "$1");
+	return authorPart.replace(/[\s\r\n]+y[\s\r\n]+/g, ',').split(',');
+}
+
+
+function extractTitle(entry) {
+	let titleRegex = new RegExp(articleNumberPrefix + '\\s+.*,\\s+'
+		 +  quotationMarks + '(.*)' + quotationMarks + '.*$', 'g');
+	let onelineInnerText = entry.innerText.replace(/[\r\n]+/gm, " ").replace(/\s\s+/gm, " ");
+	let titlePart = onelineInnerText.replace(titleRegex, "$1");
+	return cleanTitle(titlePart);
+}
+
+
+function extractURL(entry) {
+	let anchor = entry.querySelector('a');
+	let href = anchor ? anchor.href : null;
+	if (href) {
+		return href;
+	}
+}
+
+function cleanTitle(title) {
+	title = title.replace(/[\n\r]+/, '');
+	title = title.replace(/(?:^\\?")(.*)(?:\\?"$)/g, "$1");
+	title = title.replace(/\s\s+/gm, " ");
```

Comment:
Also `m` modifier not necessary... maybe you could instead write /\s{2,}/ then its a bit easier to understand, but thats a matter of taste